src/paperless/settings.py

```diff
@@ -70,6 +71,23 @@ def __get_path(key: str, default: str) -> str:
     return os.path.abspath(os.path.normpath(os.environ.get(key, default)))
 
 
+def __get_list(
+    key: str,
+    default: Optional[List[str]] = None,
+    sep: str = ",",
+) -> List[str]:
+    """
+    Return a list of elements from the environment, as separated by the given
+    string, or the default if the key does not exist
+    """
+    if key in os.environ:
+        return os.environ[key].split(sep)
```

Comment:
Be aware that `"".split(",")` is not an empty list, but `[""]` a list with an empty string as one element. Might lead to undesired side effects?

Reply:
It seems unlike to happen, but I suppose there could be the case of someone accidently adding double commas, which would result in an empty string as well.  So I updated to filter those out.